contrib/func_tests.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/zsh
+# The following script runs three known test cases for qtop, one for each of PBS, OAR, SGE
+# No output after "Testing <scheduler>" means test passed. 
+# The test actually runs qtop over known scheduler output files, and then diffs them against the expected output. 
+# While diffing, one qtop output line is omitted 
+# (the one containing word "WORKDIR"), as it contains an everchanging timestamp.
+# WARNING: this will only run in zsh, not bash
+
+export QTOPDIR="/home/sfranky/PycharmProjects/qtop"
+alias -g NF='*(.om[1])' # newest file in a directory
+
+alias sger='$QTOPDIR/qtop.py -s $QTOPDIR/contrib -FadvvO -b sge'
+alias dsger="diff <(grep -v 'WORKDIR' NF) <(grep -v 'WORKDIR' $QTOPDIR/contrib/sger_dvv_out.ref)"
```

Comment:
hm... this is a sign of the need for a command line to suppress or stabilize the output of WORKDIR line?
fi. --supress-dynamic-info-line